config/environments/development.rb

```diff
@@ -1,7 +1,7 @@
 Rails.application.configure do
   # Verifies that versions and hashed value of the package contents in the project's package.json
-    config.webpacker.check_yarn_integrity = true
-    config.webpacker.check_yarn_integrity = true  # Settings specified here will take precedence over those in config/application.rb.
+    config.webpacker.check_yarn_integrity = false
+    config.webpacker.check_yarn_integrity = false  # Settings specified here will take precedence over those in config/application.rb.
```

Comment:
Are these changes related? I think we want this to be set back to `true` so that yarn complains when package.json changes. Not sure why there were two lines though, maybe we can delete one?